stdpopsim/catalog/MusMus/demographic_models.py

```diff
@@ -0,0 +1,344 @@
+import msprime
+import numpy as np
+
+import stdpopsim
+
+_species = stdpopsim.get_species("MusMus")
+
+
+def _dom_1pop():
+    # the size during the interval times[k] to times[k+1] = sizes[k]
+    times = np.array(
+        [
+            0,
+            83,
+            180,
+            291,
+            420,
+            570,
+            743,
+            943,
+            1175,
+            1443,
+            1754,
+            2114,
+            2530,
+            3012,
+            3570,
+            4216,
+            4964,
+            5829,
+            6831,
+            7991,
+            9334,
+            10889,
+            12688,
+            14772,
+            17183,
+            19975,
+            23207,
+            26948,
+            31279,
+            36292,
+            42096,
+            48815,
+            56592,
+            65596,
+            76019,
+            88084,
+            102052,
+            118221,
+            136938,
+            158606,
+            183689,
+            212726,
+            246340,
+            285254,
+            330300,
+            382446,
+            442812,
+            512693,
+            593589,
+            687237,
+            795646,
+            921140,
+            1066418,
+            1234595,
+            1429281,
+            1654653,
+            1915544,
+        ]
+    )
+    sizes = np.array(
+        [
+            2040,
+            2040,
+            3844,
+            90428,
+            145603,
```

Comment:
check that there is no offset here. From examination of fig 3, it looks like between 300-400 generations ago the Ne was ~150K. The way the arrays are set up here, I think that Ne=145,603 is associated with the time range 420-570.

Reply:
When you say offset here do you mean something being added at the stage of plotting?

Reply:
There appears to be an inconsistency between the plot in fig 3 of Fujiwara et al 2022 and your implementation. I don't know if the problem is in fig 3 or your implementation. If I understand the implementation correctly, then  each Ne in your table is being associated with the wrong time range (shifted one range back in time). The 5th Ne (145603) is associated in your implementation with the 5th time interval (420 - 570 generations ago). However, in Fig. 3, it appears to be associated  the time range somewhere between 300 to 400 generations ago, which fits your 4th time interval.  If you remove the first element in the ``sizes`` array, this should fix it IMO.